doc/developer/frr-release-procedure.rst

```diff
@@ -80,30 +80,39 @@ FRR Release Procedure
          .
            * Your Changes Here
 
-6. Change main version number:
+6. Commit the changes, adding the changelog to the commit message. Follow all
+   existing commit guidelines. The commit message should be akin to::
 
-    - Edit :file:`configure.ac` and change version in the ``AC_INIT`` command
-      to ``<version>``
+      debian, redhat: updating changelog for new release
 
-7. Commit the changes, adding the changelog to the commit message. Follow all
-   existing commit guidelines.
+7. Create a new branch based on ``master``, cherry-pick the commit made in step
+   6, and use it to create a PR against ``master``. This way ``master`` has the
+   latest changelog for the next cycle.
 
-8. Create and submit a GitHub pull request, with the ``HEAD`` set to
-   ``stable/<version>`` and the base set to the upstream ``master`` branch.
-   Allow NetDef CI to complete its run and verify that all package builds were
-   successful.
+8. Change main version number:
 
-9. Create a git tag for the version:
+   - Edit :file:`configure.ac` and change version in the ``AC_INIT`` command
+     to ``<version>``
 
-   .. code-block:: console
+   Add and commit this change. This commit should be separate from the commit
+   containing the changelog. The commit message should be::
 
-      git tag -a frr-<version> -m "FRRouting Release <version>"
+      FRR release <version>
+
+   The version field should be complete; i.e. for ``8.0.0``, the version should
+   be ``8.0.0`` and not ``8.0`` or ``8``.
 
-10. Push the commit and new tag.
+9. Create a new branch off of ``stable/<version>``. Add a dummy commit to it
+   and create a GitHub pull request from it with the target branch set to
+   ``stable/<version>``.  This will trigger the various CI systems, which serve
+   as a sanity check on the release branch. Verify that all tests pass and that
+   all package builds are successful.
```

Comment:
instead of  doing a PR, I propose we just push to the `rc/<version>. This way the process is identical to how the stable branch would work and the CI artifacts would better mirrors the stable branch too.
```
git push origin stable/<version>:refs/head/rc/<version>
```

Reply:
So you are proposing we create an `rc/` branch and then manually trigger the CI on it?

Reply:
I already asked Martin to do this and he did. I tested 8.1 on rc/8.1 branch,. The ci triggers automatically triggers for rc/ branches

Reply:
> The ci triggers automatically triggers for rc/ branches

Did not know this, I'll make the change.

Reply:
This is new, I asked Martin for this "feature"